pymumble_py3/users.py

```diff
@@ -74,7 +74,7 @@ def update(self, message):
             actions["actor"] = message.actor
 
         for (field, value) in message.ListFields():
-            if field.name in ("session", "actor", "comment", "texture"):
+            if field.name in ("session", "actor", "comment", "texture", "plugin_context", "plugin_identity"):
```

Comment:
What are plugin_context and plugin_identity ?

Reply:
`plugin_context` is an identifier of a "space" that users share and is used by mumble to see if two players need to be spatialized relative to each other (they will be if they have the same context). It will be something like "Overwatch_Game489026".

`plugin_identity` uniquely identifies the user in-game. Typically it is the nickname of the player inside the game. I am not sure how it is used, probably mostly for logging.